reference/upgrades.md

```diff
@@ -1,5 +1,97 @@
 ## Upgrading Mayastor
 
-Mayastor v2.0.0 does not support upgrade from previous versions. The installation of the new version should be preceded by the removal of any earlier version.
+Upgrade to the latest Mayastor version is supported only from 2.1.0 or later. To upgrade from a previous version, visit [Manual steps to upgrade Mayastor](manual-steps-to-upgrade-mayastor). 
```

Comment:
Seems there's 2 upgrade files?

Reply:
This is for Civo case. But I think the link may be broken since we changed the file name in the previous PR.
@anupriya0703 ?

Reply:
@tiagolobocastro Yes, there were 2 upgrade files present, have removed one. 
@avishnu, yes, with the previous PR the link changes..updated it too.